CDP4Composition/ViewModels/ParameterRowControlViewModel.cs

```diff
@@ -199,11 +205,11 @@ protected ParameterValueSetBase GetValueSet()
 
             if (this.Parameter is ParameterOverride parameterOverride)
             {
-                valueSet = parameterOverride.ValueSet.SingleOrDefault(x => (!this.Parameter.IsOptionDependent || (x.ActualOption == this.ActualOption)));
+                valueSet = parameterOverride.ValueSet.FirstOrDefault(x => (!this.Parameter.IsOptionDependent || (x.ActualOption == this.ActualOption)));
```

Comment:
if a server is returning incomplete data, i think we fail/crash here. Perhaps wrap an exception or go back to SingleOrDefault?